fastapi_validation_i18n/scripts/publish.py

```diff
@@ -1,27 +1,81 @@
 import os
 import shutil
-import sys
+from typing import Annotated, Union
 
+import typer
+
+SUPPORTED_LOCALES = ['zh-TW', 'en-US', 'ja-JP']
+
+app = typer.Typer()
 
-def export_locale_folder(target_path: str) -> None:
-    working_dir = os.getcwd()
 
+@app.command(name='export')
+def export_locale_folder(
+    target_path: Annotated[str,
+                           typer.Argument(
+                               help='destination path to export to')] = 'locale',
+    locale: Annotated[Union[str, None],
+                      typer.Option(help='specific locale to export')] = None
+) -> None:
+    """
+    Exports either the entire locale directory or a specific locale's message.json file to the given destination path.
+
+    Args:
+    target_path: The destination directory where the locale data will be copied.
+    locale: The specific locale to export or None to export all locales.
+    """
+    working_dir = os.getcwd()
     locale_path = os.path.join(
         os.path.join(os.path.dirname(os.path.dirname(__file__))),
         'locale',
     )
 
     target_locale_path = os.path.join(working_dir, target_path)
```

Comment:
**Suggestion:** Use pathlib for path manipulations instead of os.path for better readability and simplicity.
```suggestion
    from pathlib import Path
    working_dir = Path.cwd()
    locale_path = Path(__file__).parent.parent / 'locale'
    target_locale_path = working_dir / target_path
```